.travis.yml

```diff
@@ -4,3 +4,19 @@ node_js:
   - '0.10'
   - '0.12'
   - 'iojs'
+
+cache:
+  directories:
+    - node_modules
+
+matrix:
+  fast_finish: true
+
+install:
+  # Check the size of caches
+  - du -sh ./node_modules || true
+  # Disable the spinner, it looks bad on Travis
+  - npm config set spin false
+  - npm install -g npm@latest
```

Comment:
Mandatory. 
On node v0.10.40, NPM is on v1.4.28. A error occurs with this version of NPM.
https://travis-ci.org/zckrs/generator-node/jobs/73143670